src/App.jsx

```diff
@@ -1,12 +1,88 @@
 import './App.css';
+import map from './assets/world_map.png';
+import axios from "axios";
+import {useEffect, useState} from "react";
+import sort from "./helpers/sort.jsx";
+import {regionColor} from "./helpers/region-color.jsx";
+import Search from "./pages/Search.jsx";
+import {millionFormat} from "./helpers/millionFormat.jsx";
 
 function App() {
+    const [countries, setCountries] = useState([]);
+    const [error, setError] = useState('');
+    const [searchResult, setSearchResult] = useState('')
+    const [query, setQuery] = useState('');
+
+    async function fetchData() {
+        setError('');
+        try {
+            const {data} = await axios.get('https://restcountries.com/v3.1/all');
+            setCountries(data);
+        } catch (e) {
+            // console.error(e);
+            setError("Data ophalen mislukt")
+        }
+    }
+
+    sort(countries);
+
+    async function fetchCountryData() {
+        setError('');
+        // console.log('https://restcountries.com/v3.1/name/'+query)
+        try {
+            const {data} = await axios.get('https://restcountries.com/v3.1/name/' + query);
+            if (Object.keys(data).length > 1) {
```

Comment:
Als je wilt controleren hoeveel objecten er in een array zitten, dan kun/moet je dat zo doen
```suggestion
            if (data.length > 1) {
```